framework/include/utils/FormattedTable.h

```diff
@@ -111,6 +111,11 @@ class FormattedTable
    */
   void setPrecision(unsigned int precision) { _csv_precision = precision; }
 
+  /**
+   * Sorts columns alphabetically.
+   */
+  void sortColumns() { std::sort(_column_names.begin(), _column_names.end()); }
```

Comment:
You don't actually want to sort every single time out you output. I think you should have a boolean you maintain to short circuit the sort. Only sort if something changes with the columns.